packages/sqflite/tizen/src/database_manager.h

```diff
@@ -0,0 +1,65 @@
+#ifndef DATABASE_MANAGER_H_
+#define DATABASE_MANAGER_H_
+
+#include <flutter/standard_method_codec.h>
+
+#include "list"
+#include "sqlite3.h"
+#include "string"
+
+struct DatabaseError : public std::runtime_error {
+  DatabaseError(int code, const char *msg)
+      : std::runtime_error(std::string(msg) + " (code " + std::to_string(code) +
+                           ")") {}
+};
+
+class DatabaseManager {
+  static const int BUSY_TIMEOUT_MS = 2500;
+
+ public:
+  sqlite3 *sqliteDatabase;
+  std::map<std::string, sqlite3_stmt *> stmtCache;
+  bool singleInstance;
+  std::string path;
+  int id;
+  int logLevel;
+
+  DatabaseManager(std::string aPath, int aId, bool aSingleInstance,
+                  int aLogLevel);
+  virtual ~DatabaseManager();
+
+  typedef std::variant<int64_t, std::string, double, std::vector<uint8_t>,
+                       std::nullptr_t>
+      resultvalue;
+  typedef std::vector<resultvalue> result;
+  typedef std::vector<result> resultset;
+  typedef std::vector<std::string> columns;
+  typedef flutter::EncodableList parameters;
+
+  void open();
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Names

Ordinarily, functions should start with a capital letter and have a capital letter for each new word.